project/api/migrations/0008_auto_20190331_2200.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.1.7 on 2019-03-31 22:00
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0007_auto_20190331_2129'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='cacheddata',
+            name='date',
+            field=models.DecimalField(decimal_places=1, default=0.0, max_digits=8),
```

Comment:
Do we need this migrations (to DecimalField) on master branch?

Reply:
Not strictly, since we can't cast date to int we had to set up migrations to remove the field and then re add it as int, so I was compelled to not touch the migrations. I will try and clean it up a bit.